src/pretix/base/settings.py

```diff
@@ -1836,17 +1836,24 @@ def unserialize(cls, s):
         'Mx',
         'Dr',
         'Professor',
-        'Sir'
+        'Sir',
     ))),
     ('german_common', (_('Most common German titles'), (
         'Dr.',
         'Prof.',
         'Prof. Dr.',
     )))
 ])
+
+PERSON_NAME_SALUTATIONS = [
+    pgettext_lazy("person_name_salutation", "Ms"),
+    pgettext_lazy("person_name_salutation", "Mr"),
+]
+
 PERSON_NAME_SCHEMES = OrderedDict([
     ('given_family', {
         'fields': (
+            # field_name, label, width-weight
```

Comment:
I spent a few minutes finding out what the number was :sweat_smile:

Reply:
Feel free to improve the comment :D

Reply:
well the comment wasn't there before :sweat_smile: